cost/azure/savings_realized/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.7
+
+- Issue with empty `bill_source_expressions` fixed.
```

Comment:
This isn't meaningful to anyone who isn't familiar with the policy code. Please describe the fix in terms of what a user would see when trying to use this policy and what they should see now instead.

Reply:
Description updated.